input_definition.go

```diff
@@ -254,6 +265,56 @@ type InputDefinitionInfo struct {
 	Fields []InputDefinitionField `json:"fields"`
 }
 
+// Validate the InputDefinitionInfo data
+func (i *InputDefinitionInfo) Validate(columnLabel string) error {
+	numPrimaryKey := 0
+	accountRowID := make(map[string]uint64)
+
+	if len(i.Frames) == 0 {
+		return fmt.Errorf("At least one frame is required per Input Definition")
+	}
+
+	for _, frame := range i.Frames {
+		if err := frame.Validate(); err != nil {
```

Comment:
I'd like to keep the validate in the input frame.  Then we can add a FrameOption validate and do that in one place.